assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -7102,12 +7102,15 @@ private void setNewAssignmentParameters(RunData data, boolean validify) {
                 String associatedAssignmentTitles = "";
                 // check assignments from the site
                 for (Assignment a : assignments) {
+                    if(assignmentId.equals(a.getId())){
+                        continue;
```

Comment:
A small thing. Leave a space after the if keyword, and a space before  the trailing brace.